src/common/utils/MetaKeyUtils.cpp

```diff
@@ -256,6 +258,41 @@ HostAddr MetaKeyUtils::parseMachineKey(folly::StringPiece key) {
   return MetaKeyUtils::deserializeHostAddr(key);
 }
 
+const std::string& MetaKeyUtils::hostDirPrefix() { return kHostDirsTable; }
+
+const std::string MetaKeyUtils::hostDirHostPrefix(std::string host) {
+  return kHostDirsTable + host;
+}
+
+std::string MetaKeyUtils::hostDirKey(std::string host, Port port) {
```

Comment:
Maybe a name corresponding to"parseHostDirKey" like makeHostDirKey?

Reply:
It has two reasons:
1. keep consistent with others.
2. add "make" or "get" or sth like that is a little redundant, the original xxKey is clear enough.

Reply:
> It has two reasons:
> 
> 1. keep consistent with others.
> 2. add "make" or "get" or sth like that is a little redundant, the original xxKey is clear enough.

I think using getHostDirKey is better, because you return something here, only using  xxKey is not clear.

Reply:
Just want to keep consistent with others.